src/v3/src/types/schema.ts

```diff
@@ -146,6 +146,15 @@ export type TokenReplacementValue = {
 
 export type TokenReplacement = Partial<Record<TokenSearchValue, TokenReplacementValue>>;
 
+export type RegistrationElementSchema = Modify<Input, {
+  'label-top'?: boolean;
+  placeholder?: string;
+  'data-se'?: string;
+  options?: IdxOption[] | Record<string, string>[];
+  sublabel?: string;
+  wide?: boolean;
+}>;
```

Comment:
The v2 Registration schema is just a derivative of the `Input` object with some additional fields. 

I am going to file a new ticket to respect some of these new fields, i.e. `sublabel` should display a label under the main label, but it requires changes to many components and I wanted this PR to be as small as possible.